docs/reference/ml/ml-shared.asciidoc

```diff
@@ -284,6 +283,14 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=mode]
 include::{docdir}/ml/ml-shared.asciidoc[tag=time-span]
 end::chunking-config[]
 
+tag::class-assignment-objective[]
+Defines the objective to optimize when assigning class labels. Available
+objectives are maximize_accuracy and maximize_minimum_recall. When maximizing
+accuracy class labels are choosen to maximize the number of correct predictions.
```

Comment:
I think we have to go with `chosen`

Reply:
typo

Reply:
[beaf060](https://github.com/elastic/elasticsearch/pull/52763/commits/beaf0604114c81529211a72de5db637510661438)